### PR TITLE
Fix container router bugs

### PR DESCRIPTION
**Pull request by AlfonsoUceda**

### CODE REVIEW

**Comment by joneslee85** [comment]
:+1:

**Comment by jodosha** [comment]
@AlfonsoUceda Great! :clap:

File: test/fixtures.rb [hunk 1]
```diff
@@ -255,3 +235,61 @@ def call(params)
     end
   end
 end
+
+module Back
+  class Application < Lotus::Application
+    configure do
+      routes do
+        get     '/home', to: 'home#show'
```
Comment (joneslee85) [inline test/fixtures.rb]: nitpick, can we remove the empty spaces

File: test/fixtures.rb [hunk 2]
```diff
@@ -255,3 +235,61 @@ def call(params)
...
+        include Back::Action
+
+        def call(params)
+          self.body = 'hello from Back users endpoint'
+        end
+      end
+    end
+  end
+end
+module Front
```
Comment (joneslee85) [inline test/fixtures.rb]: nitpick, add an empty line here would help the reading

File: test/integration/container_test.rb [hunk 3]
```diff
@@ -1,17 +1,48 @@
...
+
+  def app
+    @container
+  end
+
+  def response
+    last_response
+  end
+
+  it 'should reach to endpoints' do
```
Comment (joneslee85) [inline test/integration/container_test.rb]: s/should//

**Comment by joneslee85** [comment]
I think it should get rebased firstly

**Pull request merged**